lib/code_corps/stripe_service/validators/project_can_enable_donations.ex

```diff
@@ -17,8 +17,10 @@ defmodule CodeCorps.StripeService.Validators.ProjectCanEnableDonations do
 
   and only one of:
 
-  - `Organization` with a `StripeConnectAccount`, in production env, which has `charges_enabled: true`
-  - `Organization` with a `StripeConnectAccount`, not in production env
+  - `Organization` with a `StripeConnectAccount`, in production env, which
+    has `charges_enabled: true` and `transfers_enabled: true`
+  - `Organization` with a `StripeConnectAccount` and
+    `transfers_enabled: true`, not in production env
```

Comment:
I'm not sure what my original reasoning was for this (I was the one originally implementing the validator), but I'm not seeing the benefits in splitting up the environments here.

I think it may have just been for easier manual testing, so I'm considering it might be a good idea to just ditch the check for the correct environment. 

It should just be 
- `Organization` with a `StripeConnectAccount`, which has `charges_enabled: true` and `transfers_enabled: true`

I would remove the method on line 38 and update the documentation.